kratos/utilities/entities_utilities.h

```diff
@@ -0,0 +1,103 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_ENTITIES_UTILITIES)
+#define KRATOS_ENTITIES_UTILITIES
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+/**
+ * @namespace EntitiesUtilities
+ * @ingroup KratosCore
+ * @brief This namespace includes several utilities necessaries for the computation of entities functions in a efficient way
+ * @author Vicente Mataix Ferrandiz
+ */
+namespace EntitiesUtilities
+{
+    /**
+     * @brief This method initializes all the entities (conditions, elements, constraints)
+     * @param rModelPart The model of the problem to solve
+     */
+    void KRATOS_API(KRATOS_CORE) InitializeAllEntities(ModelPart& rModelPart);
+
+    /**
+     * @brief This method initializes all the conditions
+     * @param rModelPart The model of the problem to solve
+     */
+    template<class TEntityType>
+    PointerVectorSet<TEntityType, IndexedObject>& KRATOS_API(KRATOS_CORE) GetEntities(ModelPart& rModelPart);
```

Comment:
You cannot do that in windows. Try `KRATOS_API(KRATOS_CORE) PointerVectorSet<TEntityType, IndexedObject> & GetEntities(ModelPart& rModelPart)`

Same my be happening elsewhere.

Reply:
@roigcarlo looks like it does not work

Reply:
Oh you lie to me. it works. Now the error is different. The problem now is a linking problem, probably because there is no explicit instantiation on the core of the function you are exporting (for Element, Condition and MasterSlaveCondition)

Reply:
Huuum